From c94046aaf3195436fe495b5363d18a023e6aa8a7 Mon Sep 17 00:00:00 2001
From: Simon McVittie <smcv@debian.org>
Date: Fri, 20 Apr 2018 09:05:40 +0100
Subject: [PATCH 1/4] qemu: Expect a prompt after successfully running "echo
 ok" on ttyS1

If we don't wait for a prompt after every command, then we run the
risk of losing track of which prompt corresponds to which command,
and thinking a command has completed successfully when it actually
hasn't.

Signed-off-by: Simon McVittie <smcv@debian.org>
---
 virt/autopkgtest-virt-qemu | 2 ++
 1 file changed, 2 insertions(+)

diff --git a/virt/autopkgtest-virt-qemu b/virt/autopkgtest-virt-qemu
index d89536c..57a7478 100755
--- a/virt/autopkgtest-virt-qemu
+++ b/virt/autopkgtest-virt-qemu
@@ -133,6 +133,8 @@ def check_ttyS1_shell():
         return True
     except VirtSubproc.Timeout:
         return False
+    else:
+        VirtSubproc.expect(term, b'#', 10)
 
 
 def setup_shell():
-- 
2.17.0

